prove/Develop05/CheckListGoal.cs

```diff
@@ -16,15 +16,22 @@ public CheckListGoal(string name, string description, int points,string goalType
         _tries = tries;
        // _listGoal = ListGoal;
     }
-     public override void RecordEvents()
+     public override int RecordEvents()
     { _tries++;
       if(_tries == _timesToGetBonus)
-      {
-        _isComplete = true;        
+      {               
         Console.WriteLine($"Congratulations! You have earned {_points}+{_bonus} points!");
-        _points = _points + _bonus; 
+        _points = _points + _bonus;
+        _tries = 0; 
+        _isComplete = true; 
+        return _points;
                    
-      }      
+      } 
+      else
+      {
+        return 0;
+      }  
+        
```

Comment:
well in this part I realized when I created record event, it was returning points in every case, so that something wrong because the points should get when the checklist is ready, so I made a if and else, the if is when it gets the points and the else it is when it's still working on.